recipes/taocpp-taopq/all/conanfile.py

```diff
@@ -70,8 +70,12 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
-        tc.variables["TAOPQ_BUILD_TESTS"] = False
-        tc.variables["TAOPQ_INSTALL_DOC_DIR"] = "licenses"
+        if Version(self.version.replace("cci.", "")) < "20211017":
```

Comment:
As this library always uses the `cci.` schema, you can always compare directly with `Version` directly, see https://github.com/conan-io/conan/pull/15370/files